plugins/fs/src/qmlpath.cpp

```diff
@@ -0,0 +1,299 @@
+#include "qmlpath.h"
+#include <QFileInfo>
+#include <QJSValueIterator>
+#include <QDebug>
+#include <QFile>
+
+namespace lv{
+
+QmlPath::QmlPath(QObject *parent)
+    : QObject(parent)
+{
+}
+
+QString QmlPath::name(const QString &path){
+    return QFileInfo(path).fileName();
+}
+
+QString QmlPath::join(QJSValue part1, QJSValue part2)

```

Comment:
Checks are needed in case paths end or begin with `/` character. i.e.

`Path.join('path1/', 'path2')` will result in `path1//path2`